src/lib/bootstrap.js

```diff
@@ -48,7 +48,7 @@ function startVerdaccio(config: any, cliListen: string, configPath: string, pkgV
         // http
         webServer = http.createServer(app);
       }
-      if (config.server && config.server.keepAliveTimeout) {
+      if (config.server && typeof config.server.keepAliveTimeout !== 'undefined' && config.server.keepAliveTimeout !== 'null') {
```

Comment:
:-) don't like lodash 👩‍🚀  ?

Reply:
lodash, underscore and ithers are mostly too bloated. The current code should be clear enough. No need to make it more abstract or complicated imo.

Reply:
Agree, but, I admit that lodash was really useful to put the order in the house. Perhaps is a moment to introduce something smaller or homemade that does not force to write too much boilerplate.